manifests/crd/acceleratorprofiles.opendatahub.io.crd.yaml

```
@@ -28,27 +28,36 @@ spec:
               properties:
                 displayName:
                   type: string
+                  description: "The display name of the accelerator profile visible in the dashboard UI."
                 enabled:
                   type: boolean
+                  description: "Indicates whether the accelerator profile is visible and selectable in the dashboard UI."
```

```suggestion
                  description: "Indicates whether the accelerator profile is available for new resources."
```
Disabled profiles are visible in the UI during edit.